common/lib/xmodule/xmodule/modulestore/tests/django_utils.py

```diff
@@ -436,6 +436,22 @@ def create_non_staff_user(self):
         nonstaff_user.save()
         return nonstaff_user, password
 
+    def create_staff_user(self):
```

Comment:
As mentioned above, I don't think you need this method because `self.user` is already a staff user.